src/__test__/operate.test.js

```diff
@@ -0,0 +1,23 @@
+import operate from '../logic/operate';
+
```

Comment:
- [ ] Please also cover the operation % to comply with the project requirements.

Reply:
We are now covering the % operation in the project, thanks for the advice.